File: templates/template.ts
// BOJ 1000 TypeScript(node.js)
//출처: https://www.acmicpc.net/help/language 
var fs = require('fs');
var input = fs.readFileSync('/dev/stdin').toString().split(' ');
var a = parseInt(input[0]);
var b = parseInt(input[1]);
console.log(a+b);

File: templates/template.pl
#!/usr/bin/perl
# BOJ 1000 Perl
# 출처: https://www.acmicpc.net/help/language 
my $a = <STDIN>;
my @b = split(/ /, $a);

my $ans = @b[0] + @b[1];

print $ans;